opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/OpenTelemetryMetricsExporter.java

```diff
@@ -0,0 +1,198 @@
+                  + point.getTimestamp().getNanos();
+          type = metric.getMetricDescriptor().getType();
+          switch (type) {
+            case GAUGE_INT64:
+            case CUMULATIVE_INT64:
+              points.add(
+                  LongPoint.create(
+                      epochNanos,
+                      epochNanos,
+                      labels,
+                      point
+                          .getValue()
+                          .match(
+                              Double::longValue,
+                              arg -> arg,
+                              arg -> null,
+                              arg -> null,
+                              arg -> null)));
+              break;
+            case GAUGE_DOUBLE:
+            case CUMULATIVE_DOUBLE:
+              points.add(
+                  DoublePoint.create(
+                      epochNanos,
+                      epochNanos,
+                      labels,
+                      point
+                          .getValue()
+                          .match(
+                              arg -> arg,
+                              Long::doubleValue,
+                              arg -> null,
+                              arg -> null,
+                              arg -> null)));
+              break;
+            case SUMMARY:
+              points.add(
+                  SummaryPoint.create(
+                      epochNanos,
+                      epochNanos,
+                      labels,
+                      point
+                          .getValue()
+                          .match(
+                              arg -> null,
+                              arg -> null,
+                              arg -> null,
+                              Summary::getCount,
+                              arg -> null),
+                      point
+                          .getValue()
+                          .match(
+                              arg -> null, arg -> null, arg -> null, Summary::getSum, arg -> null),
+                      point
+                          .getValue()
+                          .match(
+                              arg -> null,
+                              arg -> null,
+                              arg -> null,
+                              OpenTelemetryMetricsExporter::mapPercentiles,
+                              arg -> null)));
+              break;
+            default:
+              LOGGER.warning(type + " not supported by OpenCensus to OpenTelemetry migrator.");
+              break;
+          }
+        }
+        MetricData.Type metricDataType = mapType(type);
+        if (metricDataType != null) {
+          // TODO(@zoercai): resource, attributes, instrumentationLibraryInfo need to be
+          // added/mapped properly
+          metricData.add(
+              MetricData.create(
+                  Resource.getDefault(),
+                  InstrumentationLibraryInfo.create("das", "1"),
```

Comment:
Maybe can use `io.opentelemetry.opencensusshim` as name at least for now. Let's go ahead and extract a constant for the InstrumentationLibraryInfo too